packages/grafana-runtime/src/utils/migrationHandler.ts

```diff
@@ -0,0 +1,69 @@
+import { DataQueryRequest } from '@grafana/data';
+import { DataQuery } from '@grafana/schema';
+
+import { config } from '../config';
+import { getBackendSrv } from '../services';
+
+import { DataSourceWithBackend } from './DataSourceWithBackend';
+
+export interface MigrationHandler {
```

Comment:
Since this is the only public facing part of this file, can we move the @alpha Experimental to this interface instead?